tools/engine_tool/pubspec.yaml

```diff
@@ -31,47 +26,7 @@ dependencies:
   process_runner: any
 
 dev_dependencies:
-  async_helper: any
-  expect: any
-  litetest: any
-  process_fakes:
-    path: ../pkg/process_fakes
-  smith: any
-
-dependency_overrides:
-  args:
-    path: ../../third_party/dart/third_party/pkg/args
-  async:
-    path: ../../third_party/dart/third_party/pkg/async
-  async_helper:
-    path: ../../third_party/dart/pkg/async_helper
-  collection:
-    path: ../../third_party/dart/third_party/pkg/collection
-  expect:
-    path: ../../third_party/dart/pkg/expect
-  file:
-    path: ../../third_party/dart/third_party/pkg/file/packages/file
   litetest:
     path: ../../testing/litetest
-  logging:
-    path: ../../third_party/dart/third_party/pkg/logging
-  meta:
-    path: ../../third_party/dart/pkg/meta
-  path:
-    path: ../../third_party/dart/third_party/pkg/path
-  platform:
-    path: ../../third_party/pkg/platform
-  process:
-    path: ../../third_party/pkg/process
-  process_runner:
-    path: ../../third_party/pkg/process_runner
-  smith:
-    path: ../../third_party/dart/pkg/smith
-  source_span:
-    path: ../../third_party/dart/third_party/pkg/source_span
-  string_scanner:
-    path: ../../third_party/dart/third_party/pkg/string_scanner
-  term_glyph:
-    path: ../../third_party/dart/third_party/pkg/term_glyph
-  yaml:
-    path: ../../third_party/dart/third_party/pkg/yaml
+  process_fakes:
```

Comment:
this `path` doesn't get picked up by the workspace?

Reply:
No, because `process_fakes` isn't a pub dependency (it's not on pub.dev) so there is no resolution to point to.

Reply:
filed: https://github.com/dart-lang/pub/issues/4318